servers.conf.erb

```diff
@@ -115,7 +115,7 @@ server {
 server {
   server_name pro.passculture-testing.beta.gouv.fr;
   listen <%= ENV['PORT'] %>;
-  rewrite ^/(.*)$ https://pro.testing.passculture.team/$1 permanent;
```

Comment:
tu vas peut être avoir un problème pour les redirects qui inclut déjà des query parameters :thinking:

Reply:
@pli01 tu connais nginx ?

Reply:
Et ces 2 pistes peut etre ? 

https://stackoverflow.com/questions/16532293/nginx-proxy-pass-is-it-possible-to-add-a-static-parameter-to-the-url/66703673#66703673

Soit
```
rewrite ^ https://pro.testing.passculture.team$uri?redirect=true permanent;
```

Soit avec proxy_pass
```
location / {
    rewrite ^/?(.*) /$1?redirect=true break;
    proxy_pass https://pro.testing.passculture.team$uri$is_args$args;
}
```

Reply:
Merci pour vos suggestions :pray: J'ai appliqué la modif en utilisant le rewrite.